WalletWasabi.Documentation/HowToDeploy.md

```diff
@@ -0,0 +1,25 @@
+# The agreed procedure to deploy something to production
+
+1. Create a summary of what is going to be deployed (we are still working on this and don't have anything automatic yet).
+2. Announce in the Slack Integrations channel the deploy one specific commit to testnet using the @aqui tag and share the summary.
+3. Affiliates must acknowledge the notification.
+4. Affiliates must express whether they are willing to test what has been deployed or not.
+   1. In case Affiliates **are not** interested in testing it then the Wasabi team will notify the planned deploy date to production, and that's it.
```

Comment:
```suggestion
   1. In case Affiliates **are not** interested in testing it, then the Wasabi team will notify the planned deployment date to production, and that's it.
```